akka-persistence-testkit/src/main/scala/akka/persistence/testkit/scaladsl/EventSourcedBehaviorTestKit.scala

```diff
@@ -221,9 +221,21 @@ object EventSourcedBehaviorTestKit {
   def clear(): Unit
 
   /**
-   * The underlying `PersistenceTestKit` for the in-memory journal and snapshot storage.
+   * The underlying `PersistenceTestKit` for the in-memory journal.
    * Can be useful for advanced testing scenarios, such as simulating failures or
    * populating the journal with events that are used for replay.
    */
   def persistenceTestKit: PersistenceTestKit
+
+  /**
+   * The underlying `SnapshotTestKit` for snapshot storage. Present only if snapshots are enabled.
+   * Can be useful for advanced testing scenarios, such as simulating failures or
+   * populating the storage with snapshots that are used for replay.
+   */
+  def snapshotTestKit: Option[SnapshotTestKit]
+
+  /**
+   * Initializes behavior from provided state and/or events.
+   */
+  def initialize(stateOption: Option[State], events: Event*): Unit
```

Comment:
here I think we can have two methods instead of one taking an `Option`

```scala
def initialize(initialState: State, events: Event*): Unit 
def initialize(events: Event*): Unit
```

As such we don't need to bother users in asking them to wrap it `Some` or passing `None`.

(same for the java variant)

Reply:
I though about doing it, but in the end decided just to keep it as it was in Lagom TestKit, but this approach does look more ergonomic, I will implement it.